app/assets/javascripts/atWhoAutoComplete.js

```diff
@@ -1,8 +1,14 @@
 (function() {
 
   // settings at https://github.com/ichord/At.js/wiki/Base-Document#settings
+
+  const displayName = (item) => item.username ? item.username : item.name;
+
   var at_config = {
     at: "@",
+    displayTpl: (item) => `<li>${displayName(item)}</li>`,
+    insertTpl: (item) => `@${displayName(item)}`,
+    data: '/users/active',
```

Comment:
How will this interact with the request down on line 15, of `$.getJSON("/api/srch/profiles?query=" + query + "&sort_by=recent&field=username", {}, function(data) {`? Does this override it?

Reply:
Maybe even if both can co-exist, we should add a comment clarifying the difference because it's a little hard to follow.

Reply:
aha! So `data` is the initial query after an `@`, and the `remoteFilter` is when you begin typing letters?

Reply:
@jywarren Exactly! I'll add a comment to clarify this.